feature-utils/poly-look/test/react-components/overlays/base.test.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import BaseOverlay from "../../../src/react-components/overlays/base";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+
+const testClassName = "test1";
+const testId = "testId";
+const testComponent = (
+  <BaseOverlay className={testClassName}>
+    <div data-testid={testId}></div>
+  </BaseOverlay>
+);
+
+const centeredTestComponent = (
+  <BaseOverlay className={testClassName} centered={true}></BaseOverlay>
+);
+
+it("renders correctly", () => {
```

Comment:
We usually group "it" in "describe" blocks. It does not add a lot of functionality, but it literally describes what you're testing. Take into account that this will just say "renders correctly" in the test log, without providing a context. `describe` provides that context, even if it's not really functional.
(Saving another @fhd's nit here)

Reply:
Ok this is what i thought about this:
it refers to "it" so it's "it renders correctly"
What would I put in the describe? The component.
Now we have "Baseoverlay" -> "it renders correctly"
When there is just one suite with the same name as the file I find it redundant to put describe.

Reply:
The name of the class might be removed from the actual reports in the test log. It's just a little helper.

Reply:
Ok I thought this was on purpose as it's missing in the chips. And as you are renowned duplication opponent ... :D Happy to add it